pyproject.toml

```diff
@@ -55,28 +55,30 @@ classifiers = [
 ]
 dependencies = [
     "minimega==2.9",
+    "ClusterShell<=1.9.2",
     "colorama<=0.4.6",
+    "coverage<=7.6.10",
     "grpcio>=1.49.0,<=1.67.0",
     "grpcio-tools>=1.49.0,<=1.69.0",
+    "importlib_metadata>=3.6,<=8.5.0",
     "Jinja2>=3.1.2,<=3.1.5",
     "netaddr<=1.3.0,>=0.7.0",
     "networkx>=2.3,<=3.4.2",
     "protobuf>=5.0.0,<=5.29.3",
-    "ClusterShell<=1.9.2",
     "pytest<=8.3.4",
+    "pytest-cov<=6.0.0",
     "python-dotenv<=1.0.1",
     "PyYAML<=6.0.2",
     "qemu.qmp==0.0.3",
-    "rich>=13.6.0,<13.10",
     "requests>=2.22.0,<=2.32.3",
-    "importlib_metadata>=3.6,<=8.5.0",
+    "rich>=13.6.0,<13.10",
 ]
 
 [project.optional-dependencies]
-test = [
-    "tox<=4.23.2",
-    "coverage<=7.6.10",
-    "pytest-cov<=6.0.0",
+mcs = [
+    "firewheel-repo-base",
+    "firewheel-repo-linux",
+    "firewheel-repo-vyos",
 ]
```

Comment:
We may wish to add the full set of MCs here (including DNS, NTP, layer 2, and tutorials). I only added these three since they are the minimum set required to run the full test suite (provided the appropriate images are available).